tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -39,7 +41,31 @@
 from tests_e2e.scenarios.lib.agent_test import AgentTest
 from tests_e2e.scenarios.lib.agent_test_context import AgentTestContext
 from tests_e2e.scenarios.lib.identifiers import VmIdentifier
-from tests_e2e.scenarios.lib.logging import log
+from tests_e2e.scenarios.lib.logging import log as agent_test_logger  # Logger used by the tests
+
+
+def _initialize_lisa_logger():
```

Comment:
Previously this was done in tests_e2e/scenarions/lib/logging.py as part of the setup for the global logger. Since now I separated the LISA log from the test logs, I moved this to this module, which is our interface with LISA.